admin/base/settings/defaults.py

```diff
@@ -219,3 +224,10 @@
 
 ENTRY_POINTS = {'osf4m': 'osf4m', 'prereg_challenge_campaign': 'prereg',
                 'institution_campaign': 'institution'}
+
+
+# Change desk settings in local.py
+DESK_KEY = ''
+DESK_KEY_SECRET = ''
+DESK_TOKEN = ''
```

Comment:
I believe these credentials are shared between desk admin users. We could get a set for each user that wants to deal with users on Desk. This would then be along the lines of email and password except it would be 4 fields rather than 2.

As an alternative you are welcome to discuss opening a port with Michael in the VPN.